src/Helper/SemanticVersion.php

```diff
@@ -91,8 +91,8 @@ public function bump(string $release, string $extraRelease = ''): string
                 $skipBumpRelease = true;
             }
             $extraVersion = empty($partsExtra[1]) ? 0 : $partsExtra[1];
-            if (is_numeric($extraName) && (empty($partsExtra[1]) || !is_numeric($partsExtra[1]))) {
-                $extraVersion = $partsExtra[0];
+            if (is_numeric($partsExtra[1]) && (empty($partsExtra[0]) || ! is_numeric($partsExtra[0]))) {
+                $extraVersion = $partsExtra[1];
```

Comment:
@marcocesarato I'm not entirely certain about the logic I've proposed. It occurred to me that `$extraName` will **never** be numeric (?) and that `$partsExtra[0]` will **never** be numeric. Also `$partsExtra[1]` will **always** contain the version number (?).

Reply:
Yes it is correct, this is just a mistakes, thanks for report this issue